product_specifications/DVjtom_67/readme.md

```diff
@@ -0,0 +1,149 @@
+
+        Scenario: Recieving usage stats from the Tracker for existing DataCite DOIs
+            Given that Stats Processing Service received the analytics from the Tracker
+             When the end of the month arrives.
+             Then the processor must process the analytics following the COUNTER Code of Practice for Research Data.
+             And the processor must generate a JSON SUSHI report of the stats by repository with the following metrics types:
+                unique-investigations-regular
+                unique-requests-regular
+                total-investigations-regular
+                total-requests-regular
+             And processor must send the JSON SUSHI report to the REST API '/reports'
+
+        Scenario: Recieving usage stats from the Tracker for non Datacite DOIs (transfer or malformed)
+            Given that Stats Processing Service received the analytics from the Tracker
+             When data is sent to the processor
+             And the DOI name is not in our index.
+             Then the processor must discard the data.
+             And it must return a 404 Error to the tracker.
+
+Feature: Acessing Usage Stats Processing
+
+        Scenario: Displaying Usage Stats from Tracker in the REST API
+            Given that usage stats exist for a DOI
+             When a user visits to the enpoints '/dois/{doi-name}'
+             Then the user should be able to see the usage stats counts in the response
+
+        Scenario: Displaying Usage Stats from Tracker in the GrapQL API
+            Given that usage stats exist for a DOI
+             When a user visits to the endpoint '/graphql' and makes a query for a viewsCount or downloadsCount
+             Then the user should be able to see the usage stats counts in the response
+
+        Scenario: Displaying Usage Stats from Tracker in the Data metrics badge
+            Given that usage stats exist for a DOI
+             When a user visits a page that include the Data metrics badge for that DOI
+             Then the user should be able to see the usage stats counts in the badge
+
+        Scenario: Displaying Usage Stats from Tracker in the EventData API
+            Given that usage stats exist for a DOI
+             When a user visits to the enpoints '/events/doi={doi-name}'
+             Then the user should be able to see the usage stats counts as events in the response
+
+        Scenario: Displaying Usage Stats from Tracker in Fabrica
+            Given that usage stats exist for a DOI
+             When a user visits to the Usage Tracking Dashboard Tab
+             Then the user should be able to see the usage stats in different views.
+
+        Scenario: Displaying Usage Stats from Tracker in SUSHI Reports
+            Given that usage stats exist for a Repository on a given month
+             When a user visits to the enpoint '/reports/{report-id}'
+             Then the user should be able to download a usage report (SUSHI JSON) 
+
+```
+
```

Comment:
Should we mention how long we will keep the information and which data are we going to store long term?

Reply:
I was thinking about the same. According to COUNTER we need to maintain at least current year plus the previous two-years (https://www.projectcounter.org/frequently-asked-questions-faqs-counter-code-practice-release-5/). That being said, the bibliometricians might have a different perspective as they might want to keep the data for further studies.